### ISSUE TITLE
Phase 1: Parse Plena Vortaro definitions for semantic similarity

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#73]
## Summary

Parse Plena Vortaro (PV) definitions to extract Esperanto→Esperanto semantic relationships. Words with overlapping definition roots should have similar embeddings.

## Context

From the Fundamento-Centered Training plan, Phase 1 uses PV definitions as the second-highest authority source (after Fundamento's Universala Vortaro). This is critical for solving the co-occurrence problem: semantically similar words like "hundo" and "kato" may never appear together in corpus, but share definition roots → they get similar embeddings.

### 1.1 Parse PV Definitions
- [ ] Parse `data/grammar/plena_vortaro.txt` (2.3MB, ~15K entries)
- [ ] Extract definition text and parse to get definition roots
- [ ] Output: `data/vocabularies/pv_definitions.json`

Example output structure:
```json
{
    "hundo": {
        "definition": "dommastrumata karnomanĝanto",
        "definition_roots": ["dom", "mastr", "um", "karn", "manĝ"],
        "related_entries": ["kato", "besto", "mamulo"]
    }
}
```

### 1.2 Build Definition Similarity Graph
- [ ] For each pair of roots, calculate overlap in definition roots
- [ ] Build adjacency matrix or edge list for training

### 1.3 Definition-Based Training Loss
```python
def pv_similarity_loss(root_embeddings, pv_data):
    for root, data in pv_data.items():
        definition_set = set(data["definition_roots"])
        for other_root, other_data in pv_data.items():
            if root != other_root:
                other_def_set = set(other_data["definition_roots"])
                overlap = len(definition_set & other_def_set)
                if overlap > 0:
                    target_sim = overlap / max(len(definition_set), len(other_def_set))
                    loss += similarity_loss(
                        root_embeddings[root],
                        root_embeddings[other_root],
                        target_sim
                    )
```

## Training Weight

PV definitions have weight 2.0x (second only to Fundamento at 10x).

## Related

- Epic: #65 (Fundamento-Centered Training)
- Precedes: #69 (Affix transformation training)
- See: [[Fundamento-Centered-Training]] wiki page

**Comment by marctjones** [comment]
Completed. Parsed 2,103 PV entries with definition roots.

**Output**: `data/vocabularies/pv_definitions.json`

**Script**: `scripts/training/parse_pv_definitions.py`

Statistics:
- Total entries: 2,103
- Entries with relations: 2,092
- Unique definition roots: 8,640

Each entry includes:
- Definition text
- Extracted definition roots (for similarity graph)
- Line number
- Related entries
